apps/meteor/app/livechat/server/lib/RoutingManager.ts

```diff
@@ -226,6 +227,37 @@ export const RoutingManager: Routing = {
 			}),
 		);
 
+		if (isSingleContactEnabled() && inquiry.v.contactId) {
+			const contact = await LivechatContacts.findOne({
```

Comment:
i don't see the whole "contact" being used, so a projection should be added. + types!